sound/soc/sof/sof-acpi-dev.c

```diff
@@ -117,6 +118,33 @@ static const struct sof_dev_desc sof_acpi_cherrytrail_desc = {
 	.arch_ops = &sof_xtensa_arch_ops
 };
 
+static int byt_rt5651_quirk_cb(const struct dmi_system_id *id)
+{
+	/* S3 is not supported for Minnowboards */
+	no_PM_support = true;
```

Comment:
we have a sof_acpi_debug variable, you could reclaim one bit to set a DISABLE_SUSPEND.